ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -249,6 +252,79 @@ Resources:
       Roles:
         - !Ref 'InstanceRole'
 
+  # The EC2 instance writes to it, but this is introduced for the cleanup of the parameter
+  InstanceSSMParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      # This will need to be synced with the `packer-rstudio` project
+      Name: !Sub "/service-catalog/synapse/cred/${LinuxInstance}/odic-accesstoken"
+      Type: String
+      Value: to be replaced by EC2
+
+  SSMParameterEncryptionKey:
+    Type: AWS::KMS::Key
+    Properties:
+      Description: key used to encrypt SSM Parameters for a EC2 instance
+      # number of days to wait before key gets deleted
+      PendingWindowInDays: 7
+      KeyPolicy:
+        Version: "2012-10-17"
+        Statement:
+        - Sid: "Deny decrypt to all Principals that shouldn't require access"
+          Effect: 'Deny'
+          NotPrincipal:
+            AWS:
+              - !ImportValue
+                'Fn::Sub': '${AWS::Region}-sc-ec2vpc-launchrole-LaunchRoleArn'
+          Action:
+            - kms:Decrypt
```

Comment:
I tried denying `kms:*`, but it caused cloudformation to error stating that it would prevent me from modifying the policies on this Key once it's been created

Denying `kms:Decrypt` was sufficient to disallow the scipool admin user from getting the decrypted JWT